x-pack/plugin/analytics/src/main/java/org/elasticsearch/xpack/analytics/rate/RateAggregatorFactory.java

```diff
@@ -44,15 +45,21 @@
     static void registerAggregators(ValuesSourceRegistry.Builder builder) {
         builder.register(
             RateAggregationBuilder.REGISTRY_KEY,
-            List.of(CoreValuesSourceType.NUMERIC, CoreValuesSourceType.BOOLEAN),
-            RateAggregator::new,
+            Collections.singletonList(CoreValuesSourceType.NUMERIC),
```

Comment:
Is it a BWC break that we're dropping boolean here?

Reply:
Technically, yes. But this behavior was never documented, and it doesn't make any sense in this context. I am not even sure what would be an interpretation of this number in case of boolean field.